package.json

```diff
@@ -125,7 +125,9 @@
   "types": "index.d.ts",
   "typesVersions": {
     "<3.9": {
-      "*": "ts3.4/*"
+      "*": [
+        "ts3.4/*"
+      ]
```

Comment:
Was the old config wrong?

Reply:
It was not; what it was is *easy to mess up*. Doing it this way Kees things correct more easily. I plan to document this in the SemVer for Addon Types RFC and hopefully get it codified into tooling early October.